packages/curve-ui-kit/src/shared/api/figma-tokens/tokens/curve-figma-design.tokens.json

```diff
@@ -0,0 +1,6103 @@
+{
```

Comment:
6kb of JSON is too much for the bundle :scream: 
can't we extend the script to extract only what we need?

Reply:
During building, only parts of the JSON are used? Is the bundler really that smart?

Reply:
Next.js has a really strong tree shaking system and cuts off unused modules.